lib/matplotlib/axes/_axes.py

```diff
@@ -2025,6 +2025,8 @@ def make_iterable(x):
                 bottom = [0]
 
             nbars = len(left)
+            if len(height) == 1:
```

Comment:
4 of them can (I've done that below), but one of the if statements has to stay I think (as it's different for vertical or horizontal orientations)